boards/arm/mps2_an521/mps2_an521_ns.dts

```diff
@@ -101,16 +105,22 @@
 		#size-cells = <1>;
 		ranges;
 
-		/* The memory regions defined below must match what the TF-M
+		/* This code memory region must match what the TF-M
 		 * project has defined for that board - a single image boot is
 		 * assumed. Please see the memory layout in:
-		 * https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git/tree/platform/ext/target/mps2/an521/partition/flash_layout.h
+		 *
+		 * https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git/tree/platform/ext/target/arm/mps2/an521/partition/flash_layout.h
```

Comment:
Perhaps better to point to Zephyr's fork, which is what we actually use at compile time:

```suggestion
		 * https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/master/platform/ext/target/arm/mps2/an521/partition/flash_layout.h
```